src/Elastic.Apm/Model/Span.cs

```diff
@@ -186,6 +164,20 @@ private Span(double duration, string id, string name, string parentId)
 		[JsonProperty("parent_id")]
 		public string ParentId { get; set; }
 
+		/// <summary>
+		/// This holds the raw stack trace that was captured when the span either started or ended (depending on the parameter
+		/// passed to the .ctor)
+		/// This will be turned into an elastic stack trace and sent to APM Server in the <see cref="StackTrace" /> property
+		/// </summary>
+		[JsonIgnore]
+		internal StackTrace RawStackTrace;
```

Comment:
Does this need the `[JsonIgnore]`, as a field?

Reply:
No - removed.